core/node_path.cpp

```diff
@@ -1,5 +1,5 @@
 /*************************************************************************/
-/*  path_db.cpp                                                          */
+/*  node_path.cpp                                                          */
```

Comment:
Please also fix the alignment of the ending `*/` so that the "box" looks good ;) (also in the other file).